### PR TITLE
Feat/participant indicator

### PR DESCRIPTION
**Pull request by izruff**
### Description:

Adds cursor display for other users in the same session

- Added handling of cursor position updates and passing user information to the backend
- Fixed websocket reconnecting bugs by reloading after every reconnect

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "event-emitter-es6": "^1.1.5",
     "logdown": "^3.3.1",
     "reconnecting-websocket": "^4.4.0",
+    "rich-text": "^4.1.0",
```
Comment (RichDom2185) [inline package.json:40]: Is this dependency still needed? I don't see it being imported anywhere

Comment (izruff) [reply package.json:41]: No, it was supposed to be removed. I guess I forgot to remove it here.

Comment (RichDom2185) [reply package.json]: Resolved in 18964cfa10dac2bd824d2bfffca128646e17fd8f

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**